addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
Personally, I think it's much clearer what:
```js
Ember.Component.extend(PropTypesMixin, {
```

Is giving me than

```js
FrostComponent.extend({
```

If `FrostComponent` were somehow named something like `EmberComponentWithPropTypes` it might be clear in both cases.